en/getting-started/dashboard/index.md

```diff
@@ -49,9 +49,9 @@ This tab lists all the editor versions that have been downloaded and installed.
 
 ![Dashboard](index/dashboard-editor.png)
 
-Note that when you first run Cocos Dashboard, this list is empty. You can click the two buttons in the bottom right corner to import existing Creator editors locally or download and install them directly.
+> **Note**: when Cocos Dashboard is first run, this list is empty. Click the two buttons in the bottom right corner to import existing Creator editors locally or download and install them directly.
 
-- **Locate**: Add a local editor to the **Editor** version list.
+- **Locate**: Add a local editor to the **Editor** version list. Since version **v1.0.13**, allows dragging and droping the local Creator Editor into the **Editor** version list.
```

Comment:
```suggestion
- **Locate**: Add a local editor to the **Editor** version list. Since **v1.0.13**, allows dragging and droping the local Creator Editor into the **Editor** version list directly.
```